.github/workflows/wheels.yml

```diff
@@ -130,15 +130,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: actions/setup-python@v5
+      - uses: Quansight-Labs/setup-python@v5
```

Comment:
fwiw this is now supported by `actions/setup-python`

> FYI if anyone has been using quansight-labs/setup-python on github actions for free-threaded python CI or builds, action/setup-python finally released free-threaded support and we are winding down the fork - I’m going to send in PRs later today for public repos I can find on github search. 

https://discord.com/channels/1209263839632424990/1211072686545960980/1354069878973399183

https://github.com/PyO3/pyo3-async-runtimes/pull/38